src/core/call.js

```diff
@@ -11,16 +11,13 @@
  * @param token
  * requiresAuthentication: boolean
  */
-export const call = (method, path, urlParams, body, token) => {
-  const obj = {
-    method,
-    path
-  }
-  if (urlParams) { obj.urlParams = urlParams }
-  if (body) { obj.body = body }
-  if (token) { obj.token = token }
-  return obj
-}
+export const call = (method, path, urlParams, body, token) => ({
+  method,
+  path,
+  ...urlParams,
+  ...body,
+  ...token
```

Comment:
mmm.. I didn't know that the spread op was also used for including individual fields.
Shouldn't this spread all properties of "body" for example flattened in the result object, instead of including "obj.body = body" ?